omero/sysadmins/whatsnew.rst

```diff
@@ -10,6 +10,9 @@ What's new for OMERO 5.4 for sysadmins
 
 - A walkthrough has been added for :doc:`unix/server-debian9-ice36`.
 
+- A walkthrough has been added to :doc:`public`, based on a concrete
+  use-case in Dundee.
```

Comment:
As there is no summary list of links on the ``Publishing data using OMERO.web`` page unlike on the other pages to which the What's new points I would add some hint about how to look for it on the page, something like the name of the paragraph of the walkthrough itself. This will of course depend on how the paragraph will be named as discussed in https://github.com/openmicroscopy/ome-documentation/pull/1830/files#r162590029